lib/protocol/keyParser.js

```diff
@@ -1390,6 +1390,28 @@ function parseDER(data, baseType, comment, fullType) {
   return new OpenSSH_Public(fullType, comment, pubPEM, pubSSH, algo);
 }
 
+function tryParseAsCert(data, baseType, comment) {
+  if (!isSupportedCertType(baseType))
+    return new Error(`Unsupported OpenSSH cert type: ${baseType}`);
+
+  let algo;
+  let pubPEM = null;
```

Comment:
This is never assigned. Is this intentional? 

```suggestion
  const pubPEM = null;
```